### ISSUE TITLE
Error on Transfer Rewards

### ISSUE THREAD

**Issue by agulab** [PIVX-Project/PIVX-SPMT#5]
Hi there, you're doing an amazing job. I've been using SPMT since version 0.1.1 and its improvements are great.
I just run into an error while trying to Transfer some of the Rewards available, 37 out of 42, and got this error again and again after the signing process on my Ledger completes at 100%: 

```
2018-12-04 15:28:48 : Amount to send :100.98527888
EXCEPTION
version : 0.4.0-a
caller : signTxFinish
function : FinishSend
err: -342: non-JSON HTTP response with '400 Bad Request' from server
===> Exception in RPC client
EXCEPTION
version : 0.4.0-a
caller : signTxFinish
function : FinishSend
err: ("'NoneType' object has no attribute 'get'",)
===> Exception in FinishSend
```

I tried it with my local Windows wallet v3.1.1 (64-bit) and with your Amsterdam node and the result was the same.
My SPMT version is 0.4.0-a. I tried it on 0.3.1-c and run into problems too, but different ones: could connect to both wallet and ledger but Masternode couldn't be found and rewards list never fills.

**Comment by agulab** [comment]
I made one more test selecting ALL 42 rewards, and the signing process completed successfully. I didn't broadcast the transaction though, as I need to transfer only 37 of the rewards.
So after that I tried one more time with the same 37 rewards as before, and it worked! Could the 42 rewards test have changed something that made it work?
I suspect there's a bug hidden there, but you can close this if you want anyway.
Thanks for the great job.

**Comment by random-zebra** [comment]
Hey @agulab thank you so much for your feedback.
I’m glad you like the tool.

Yes, I was aware of that bug.
It should be fixed in the latest version `v0.4.1`. 
Else, please let me know, I’ll leave this issue open for a while.
